compiler/src/dotty/tools/dotc/core/Types.scala

```diff
@@ -1276,7 +1276,10 @@ object Types {
      *  except for TypeRefs where the upper bound is returned, and HKApplys,
```

Comment:
This comment is no longer accurate since this isn't limited to TypeRef anymore, right?